include/zephyr/modem/backend/stat.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024 Trackunit Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <zephyr/kernel.h>
+
+#ifndef ZEPHYR_MODEM_PIPESTAT_
+#define ZEPHYR_MODEM_PIPESTAT_
+
+/**
+ * @cond INTERNAL_HIDDEN
+ */
+
+/** Modem backend statistics instance structure */
+struct modem_backend_stat {
```

Comment:
I propose to change the name to `modem_backend_stats` as plain `stat` could also be referring to `status` for example. Also in networking we call it stats so it easy to understand what we are talking about.

Reply:
Name changed to `modem_stats` and moved out of `backends`